config/locales/en.yml

```diff
@@ -657,14 +663,10 @@ en:
     request_amends:
       heading: Request amends
     respond:
-      heading: Marking it as sent
-      reminders_html: |
-        <strong>Make sure you have:</strong>
-        <ul class="list list-bullet">
-          <li>cleared the response with the Deputy Director</li>
-          <li>uploaded the response and any supporting documents</li>
-          <li>sent the response to the person who made the request</li>
-        </ul>
```

Comment:
Why is the reminders key removed?

Reply:
it wasn't in the designs and we decided to make the FOI and ICO page the same

Reply:
🤔  Still think this would be important for KILOs and FOI.

Reply:
I think that this is a question for design. Once Mia is back we can discuss it with her, but I think keeping it to her designs for now is the best thing, we can always add it back in

Reply:
Mia agrees that we should keep "Make sure.." copy just for FOI.

![image](https://user-images.githubusercontent.com/3441519/43711124-a60201aa-9969-11e8-82db-5c76bb0e7162.png)

Ignore the action buttons in this screenshot it should have one green and one link.

Reply:
I think if we're going to change the designs on this ticket we should do it as part of another ticket

Reply:
I've just added this copy back to the page and added a spec. The reason is that this change to FOI wasn't actually defined in the original ticket but it did make the ticket a little easier to implement.